### PR TITLE
Fix: PHPCS Issues

### PR DESCRIPTION
**Pull request by anisAronno**

### CODE REVIEW

File: src/Insights.php [hunk 1]
```diff
@@ -530,14 +542,14 @@ private static function get_server_info() {
         $server_data = array();
 
         if ( isset( $_SERVER['SERVER_SOFTWARE'] ) && ! empty( $_SERVER['SERVER_SOFTWARE'] ) ) {
-            $server_data['software'] = $_SERVER['SERVER_SOFTWARE'];
+            $server_data['software'] = sanitize_key( wp_unslash( $_SERVER['SERVER_SOFTWARE'] ) );
```
Comment (tareq1988) [inline src/Insights.php:545]: No need to sanitize here

File: src/Client.php [hunk 2]
```diff
@@ -232,16 +228,18 @@ public function send_request( $params, $route, $blocking = false ) {
...
-            'timeout'     => 30,
-            'redirection' => 5,
-            'httpversion' => '1.0',
-            'blocking'    => $blocking,
-            'headers'     => $headers,
-            'body'        => array_merge( $params, array( 'client' => $this->version ) ),
-            'cookies'     => array()
-        ) );
+        $response = wp_remote_post(
+            $url, array(
```
Comment (tareq1988) [inline src/Client.php:232]: Indentation problem here. I see the spaces have been replaced with tabs. Add a `.editorconfig` file in the root. [Example](https://github.com/weDevsOfficial/wp-user-frontend/blob/develop/.editorconfig)

```
root = true

[*]
indent_style = space
indent_size = 4
end_of_line = lf
charset = utf-8
trim_trailing_whitespace = true
insert_final_newline = true

[*.md]
trim_trailing_whitespace = false

[{.jshintrc,*.json,*.yml}]
indent_style = space
indent_size = 2

[{*.txt,wp-config-sample.php}]
end_of_line = lf
```

File: src/Insights.php [hunk 3]
```diff
@@ -210,16 +208,19 @@ public function send_tracking_data( $override = false ) {
...
 
-        $users = get_users( array(
-            'role'    => 'administrator',
-            'orderby' => 'ID',
-            'order'   => 'ASC',
-            'number'  => 1,
-            'paged'   => 1,
-        ) );
+        $users = get_users(
+            array(
```
Comment (tareq1988) [inline src/Insights.php:212]: Same indentation problem

File: src/Insights.php [hunk 4]
```diff
@@ -591,34 +603,37 @@ private function get_all_plugins() {
...
+                $active_plugins[ $k ] = $formatted;
             } else {
-                $plugins[$k] = $formatted;
+                $plugins[ $k ] = $formatted;
             }
         }
 
-        return array( 'active_plugins' => $active_plugins, 'inactive_plugins' => $plugins );
+        return array(
+			'active_plugins' => $active_plugins,
```
Comment (tareq1988) [inline src/Insights.php:634]: Indentation, replace tabs with spaces.

**Review by tareq1988 — CHANGES_REQUESTED** [review]

File: src/Insights.php (hunk 1 above)
Comment (anis3139) [reply src/Insights.php:545]: If I remove that, PHPCS gives an error.

**Comment by tareq1988** [comment]
There are still tab issues, [check here](https://github.com/Appsero/client/pull/25/files) and fix.

**Comment by saiftheboss7** [comment]
@tareq1988 bhai if all good, we can merge

**Pull request merged**